src/pptr/PPTRProduct.js

```diff
@@ -102,6 +114,8 @@ export class PPTRProduct extends App.Product {
       const match = release.releaseNotes.match(/Chromium\s+(\d+\.\d+.\d+.\d+)\s*\((r\d{6})\)/i);
       if (match)
         release.chromiumVersion = `Chromium ${match[1]} (${match[2]})`;
+      else
```

Comment:
Seems like we could try to match the new pattern here before falling back to N/A. That way we wouldn’t have to hardcode the above versions. WDYT?

Reply:
Ah, you're right, we'd still need it to map the revisions to the exact version. Seems fine to do it this way then + change `CONTRIBUTING.md` to include the Chromium version in the commit message as well 👍